dask/dataframe/tests/test_dataframe.py

```diff
@@ -5579,22 +5579,6 @@ def test_attrs_series():
     assert s.fillna(1).attrs == ds.fillna(1).attrs
 
 
-@pytest.mark.xfail(
-    not PANDAS_GT_150 or pd.options.mode.copy_on_write is False,
-    reason="df.iloc[:0] does not keep the series attrs without CoW",
-)
-def test_attrs_series_in_dataframes():
-    df = pd.DataFrame({"A": [1, 2], "B": [3, 4], "C": [5, 6]})
-    df.A.attrs["unit"] = "kg"
-    ddf = dd.from_pandas(df, 2)
-
-    # Fails because the pandas iloc method doesn't currently persist
-    # the attrs dict for series in a dataframe. Dask uses df.iloc[:0]
-    # when creating the _meta dataframe in make_meta_pandas(x, index=None).
-    # Should start xpassing when df.iloc works. Remove the xfail then.
-    assert df.A.attrs == ddf.A.attrs
```

Comment:
No, but doing

```
df.A.attrs = ...
```

shouldn't work. ``df.A`` is a temporary object in a sense which means that it's metadata should not propagate to ``df``.

Reply:
Some more context: This works because pandas uses caching under the hood so that

```
df.A
df.A
``` 

returns the same object. This is an implementation detail that we should not rely on (it will also change when Copy-on-Write becomes the default).